test/evergreen/code_coverage_analysis.py

```diff
@@ -22,23 +23,84 @@ def read_timing_data(timing_data_path: str):
         print("Timing data: {} to {}, delta {}".format(start_time_secs, end_time_secs, delta_secs))
         return delta_secs
 
+# Calculate the per component coverage from the detailed files.
+def get_component_coverage_list(branch_coverage, coverage_files, outfile):
+    component_dict = defaultdict(list)
+    new_component_dict = {}
+
+    if outfile is None:
+        outfile='atlas_out_code_coverage.json'
```

Comment:
This can be achieved by setting outfile as a [keyword argument](https://www.educative.io/answers/what-are-keyword-arguments-in-python).

Reply:
Fixed